docs/roadmap.md

```diff
@@ -0,0 +1,27 @@
+# Roadmap
+
+The list is based on the survey results and the discussion in wg-etcd-operator. It is not set in stone and may be adjusted as needed.
+
+## v0.1.0
+- Create a new etcd cluster, e.g 3 or 5 members cluster
+  - Users should be able to set at least the etcd version and cluster size
+- Understand health of a cluster
+- Enabling TLS communication
+  - Should also support certificate renewal
+
+## v0.2.0
+- Upgrade across patches or one minor version
+- Scale in and out, e.g 3 members to 5 members or vise versa
+- Support customizing etcd flags
```

Comment:
```suggestion
- Support customizing etcd options (via flags or env vars)
```

Reply:
This is just roadmap. `via flags or env vars` is design & implementation level decision.

But anyway updated per your suggestion. We can adjust the roadmap when needed in future.